core/comment.js

```diff
@@ -260,8 +253,8 @@ Blockly.Comment.prototype.setVisible = function(visible) {
  * Show the bubble. Handles deciding if it should be editable or not.
  * @private
  */
-Blockly.Comment.prototype.createBubble_ = function() {
-  if (!this.block_.isEditable() || Blockly.utils.userAgent.IE) {
+Comment.prototype.createBubble_ = function() {
+  if (!this.block_.isEditable() || IE) {
```

Comment:
Yes, as @alschmiedt notes above, I think this line is easier to understand if it reads:
```
  if (!this.block_.isEditable() || userAgent.IE) {
```
(Indeed, it could be even more readable if the constant were renamed: `userAgent.isIE`.)

Reply:
Updated accordingly. Renaming the constant SGTM as well, but is beyond the scope of this PR.